Makefile

```diff
@@ -89,7 +89,21 @@ __check_var = \
     $(if $(value $1),, \
       $(error Undefined variable $1$(if $2, ($2))))
 
+check-and-create-namespace:
+	NAMESPACE="system"; \
+	EXIST=$$(kubectl get ns $$NAMESPACE 2>&1); \
+	if echo "$$EXIST" | grep -q "NotFound"; then \
+		echo "namespace system does not exists, creating one..."; \
+		kubectl create ns $$NAMESPACE; \
+	else \
+		echo "namespace system exists"; \
+	fi
+
 build-test-binaries:
 	mkdir -p ${MAKEFILE_PATH}build
 	find ${MAKEFILE_PATH} -name '*suite_test.go' -type f  | xargs dirname  | xargs ginkgo build
 	find ${MAKEFILE_PATH} -name "*.test" -print0 | xargs -0 -I {} mv {} ${MAKEFILE_PATH}build
+
+apply-dependencies: check-and-create-namespace
+	bash ${MAKEFILE_PATH}/scripts/test/install-cert-manager.sh
+	kubectl apply -f ${MAKEFILE_PATH}/config/certmanager/certificate.yaml
```

Comment:
IIUC this should be installed automatically as part of the kustomize command, and thus included in `make apply.
```
❯ kustomize build config/default | grep -C5 Certificate
# Warning: 'bases' is deprecated. Please use 'resources' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
# Warning: 'vars' is deprecated. Please use 'replacements' instead. [EXPERIMENTAL] Run 'kustomize edit fix' to update your Kustomization automatically.
        secret:
          defaultMode: 420
          secretName: webhook-server-cert
---
apiVersion: cert-manager.io/v1
kind: Certificate
metadata:
  name: vpc-resource-serving-cert
  namespace: kube-system
spec:
  dnsNames:
```

Reply:
double-checked, after installing the cert-manager, the certificate will be installed during the kustomize cmd.